subcommittee/coding-guidelines/initiatives/safe-use-of-unsafe-guidelines/learn_unsafe_rust/src/core_unsafety/intrinsics.md

```diff
@@ -0,0 +1,15 @@
+
+# Intrinsics
+
+Rust provides a set of compiler intrinsics as part of `core` and `std` respectively (https://doc.rust-lang.org/std/intrinsics/index.html).
```

Comment:
At a minimum I'd expect this chapter to go into some depth about what an intrinsic _is_, and how it is different from a regular function.